README.md

```diff
@@ -5,37 +5,44 @@
 
 ___
 
-### User documentation
+### Getting Started
 
 For user documentation, see [Add a CSI Camera as a Modular Resource](https://docs.viam.com/extend/modular-resources/examples/csi/).
 
-### usage
+You can view more platform specific details at [JETSON.md](./doc/JETSON.md) and [PI.md](./doc/PI.md).
+
+_Note: On a Raspberry Pi, you must install GStreamer plugins before running the module._
 
-1. Download appimage from releases page.
 ```bash
-sudo wget http://packages.viam.com/apps/csi-camera/viam-csi-latest-aarch64.AppImage -O /usr/local/bin/viam-csi
+sudo apt install libgstreamer-plugins-base1.0-dev 
```

Comment:
If users only need to install `base1` why does our makefile install `good/bad1`? We should probably make our setups and the docs users will use consistent, so we can catch any errors they may have. Here and in PI.md

Reply:
Fixed gstreamer dependency instructions here to include base, good and bad plugin sets.